include/map.h

```diff
@@ -109,6 +109,18 @@ typedef struct map_textures {
     SDL_Texture* p_texture_barrier;
 } map_textures_t;
 
+typedef struct map_textures_multi {
```

Comment:
Can't you add the new textures to the previous one and use the same texture?

Reply:
Yes I agree